.github/workflows/test_development_versions.yml

```diff
@@ -41,4 +41,4 @@ jobs:
         shell: bash
         run: |
           pver=${{ matrix.python-version }}
-          tox -epy${pver/./},py${pver/./}-notebook
+          tox -epy${pver/./},py${pver/./}-notebook -- --ignore-glob=*forging*serverless*
```

Comment:
@garrison is this the proper way to mute a test broken due to external dependency? Once in each workflow?

Reply:
Since it's broken locally too (not only in CI), I think it should be specified once, on [this line](https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/blob/main/tox.ini#L38), instead.